### ISSUE TITLE
Stop using browser modals

### ISSUE THREAD

**Issue by andrewda** [sakurity/securelogin#29]
Currently, at the double check milestones (2, 4, 8, 16) the user is prompted to enter their password into a plain text JavaScript prompt. We need to find a better way to do this because we don't want the user entering their password in a plain, non obfuscated text field.

**Comment by download13** [comment]
Using the `autocomplete="off"` attribute on an input field is supposed to prevent the browser from caching it's contents.

A check box could be used to make the password visible to the user if they want. This is the method used in Keepass for example.

**Comment by andrewda** [comment]
@download13 Yea, that sounds good.

**Comment by homakov** [comment]
Yes, in fact we can call this issue to get rid of using any browser modals as they are ugly.
